libsignal-service/src/models.rs

```diff
@@ -69,7 +68,6 @@ impl Contact {
             color: contact_details.color.clone(),
             verified: contact_details.verified.clone().unwrap_or_default(),
             profile_key: contact_details.profile_key().to_vec(),
-            blocked: contact_details.blocked(),
```

Comment:
I find this a little interesting, but it's not up to us :shrug:

Reply:
I'd guess there's a new way to communicate blocked users, but haven't look into it. I'll pull it in, thanks!